service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -61,6 +62,7 @@
 import bio.terra.workspace.service.workspace.model.GcpCloudContext;
 import bio.terra.workspace.service.workspace.model.OperationType;
 import bio.terra.workspace.service.workspace.model.WsmApplication;
+import com.google.api.services.notebooks.v1.model.AcceleratorConfig;
```

Comment:
We should not serialize the Google datatype for AcceleratorConfig. Suggest you specify a record for that using types that properly serialize. You can make to/fromGcp methods in there.

We do not want a serialized data dependency on an externally defined structure. If that structure changes we will have a compatibility problem outside of our control and a custom serdes that doesn't work.